core/drivers/fabric-driver/server/events.ts

```diff
@@ -55,15 +83,7 @@ function subscribeEventHelper(
         console.log(`Sending to the relay the eventSubscription error Ack: ${JSON.stringify(ack_send_error.toObject())}`);
         // Sending the fabric state to the relay.
         client.sendDriverSubscriptionStatus(ack_send_error, relayCallback);
-    });
-}
-
-function unsubscribeEventHelper(
-    call_request: eventsPb.EventSubscription,
-    client: events_grpc_pb.EventSubscribeClient
-) {
-    const newRequestId = call_request.getQuery()!.getRequestId();
-    deleteEventSubscription(call_request.getEventMatcher()!, newRequestId).then((deletedSubscription: eventsPb.EventSubscription) => {
+    } else {
```

Comment:
Don't we need to unregister the listener before sending successful unsubscription?

Reply:
Done. I'm logging warnings instead of failing, because listeners being active even when subscriptions have been deleted from the database is not a critical error; it's just a noop.